docs/docsite/rst/user_guide/playbooks_loops.rst

```diff
@@ -334,6 +334,25 @@ To control the time (in seconds) between the execution of each item in a task lo
       loop_control:
         pause: 3
 
+To avoid pausing after skipped tasks add ``pause_after_skipped_tasks: no`` to ``loop_control``::
+
```

Comment:
Please use a code-block here. It's safer for potential translation efforts on the docs.

Reply:
done